mkdocs.yml

```diff
@@ -31,8 +31,8 @@ markdown_extensions:
 
 nav:
   - Home: index.md
+  - How it works: how_it_works.md
   - Using HyP3:
-      - getting_started.md
```

Comment:
Consider deleting `getting_started.md` from the repository?  Or is it being left intentionally so it will still load at the old URL, even if it's not included in the site map?

Reply:
woops! I put it in the redirects -- just forgot to remove